subscribed_topics:
  point_cloud_internal_topic: synchronized/depth/points
  depth_candidate_holes_topic: synchronized/depth/candidate_holes
  rgb_candidate_holes_topic: synchronized/rgb/candidate_holes
  thermal_candidate_holes_topic: thermal/candidate_holes

published_topics:
  synchronizer_unlock_topic: hole_fusion/unlock/synchronizer
  processor_log_topic: processor_log
  hole_detector_output_topic: /vision/holes_direction_alert
  enhanced_holes_topic: /vision/enhanced_holes
  enhanced_images_topic: /vision/enhanced_images
  interpolated_depth_topic: /vision/interpolated_depth_image
  make_synchronizer_subscribe_to_input: subscribe_to_input
  make_synchronizer_leave_subscription_to_input: leave_subscription_to_input

  #Command line usage: image_view {namespace}/debug_valid_holes_image _image_transport:=compressed
  debug_valid_holes_image: debug_valid_holes_image

  #Command line usage: image_view {namespace}/debug_respective_holes_image _image_transport:=compressed
  debug_respective_holes_image: debug_respective_holes_image
